template/studio/schemas/plugs/index.js

```diff
@@ -3,7 +3,7 @@ export { default as infoRows } from './infoRows'
 export { default as textWithIllustration } from './textWithIllustration'
 export { default as illustration } from './illustration'
 export { default as ctaColumns } from './ctaColumns'
-export { default as ctaPlug } from './cta'
+export { default as ctaPlug } from './ctaPlug'
```

Comment:
Did you verify this change?

Reply:
As in, I wonder how this could possibly have worked before unless the files are duplicates?

Reply:
It‘s didn’t break it at least? (as in, this code works)